example/mavlink_ftp/mavlink_ftp.cpp

```diff
@@ -0,0 +1,253 @@
+/**
+ * @file fly_mission.cpp
+ *
+ * @brief Demonstrates how to Add & Fly Waypoint missions using the MAVSDK.
```

Comment:
That would need to be updated :slightly_smiling_face:

Reply:
Updated!